Lib/urllib/parse.py

```diff
@@ -773,7 +773,9 @@ def parse_qsl(qs, keep_blank_values=False, strict_parsing=False,
         def _unquote(s):
             return unquote_plus(s, encoding=encoding, errors=errors)
     else:
-        qs = bytes(qs)
+        if not qs:
+            return []
+        qs = bytes(memoryview(qs))
```

Comment:
Is it worth having a comment why going through `memoryview` first is needed (to disallow stuff like `bytes(10)`?), or is that trick idiomatic in standard library code?

Reply:
No, it is not so common. Added a comment.